operator/operator.make

```diff
@@ -1,15 +1,15 @@
-OPERATOR_SDK_VERSION=v1.0.0
+OPERATOR_SDK_VERSION=1.2.0
 
 # download operator-sdk binary
 _work/bin/operator-sdk-$(OPERATOR_SDK_VERSION):
 	mkdir -p _work/bin/ 2> /dev/null
 #	Building operator-sdk from sources as that needs below fixes:
 #     https://github.com/operator-framework/operator-sdk/pull/3787
 #     https://github.com/operator-framework/operator-sdk/pull/3786
-#   curl -L https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o $(abspath $@)
+#   curl -L https://github.com/operator-framework/operator-sdk/releases/download/v$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o $(abspath $@)
 	tmpdir=`mktemp -d` && \
 	trap 'set -x; rm -rf $$tmpdir' EXIT && \
-	git clone --branch 1.0.0+fixes https://github.com/avalluri/operator-sdk.git $$tmpdir && \
+	git clone --branch $(OPERATOR_SDK_VERSION)+fixes https://github.com/avalluri/operator-sdk.git $$tmpdir && \
```

Comment:
Do we still need additional fixes? Please add links to pending PRs for all of them so that we can check progress.

Reply:
The pending PR link(s) are available at few lines above in make target.

Reply:
Looking at https://github.com/operator-framework/operator-sdk/pull/3786 I see that it was closed as "open for a while without interation". It looks like there was a test failure that didn't get addressed for two months?

Can you do what was suggested in https://github.com/operator-framework/operator-sdk/pull/3786#issuecomment-719659176 and create an issue? Then if you have a working fix, create a PR.

Reply:
Please work with upstream and add a link to that work (probably the new issue) here.

Reply:
Reported the [issue](https://github.com/operator-framework/operator-sdk/issues/4293).